test/teams.e2e-spec.ts

```diff
@@ -0,0 +1,189 @@
+import { Test, TestingModule } from "@nestjs/testing";
+import { INestApplication, ValidationPipe } from "@nestjs/common";
+import * as request from "supertest";
+import { AppModule } from "../src/app.module";
+import { PrismaService } from "../src/prisma/prisma.service";
+import { seed } from "../prisma/seed/seed";
+import { loginAndGetTokens } from "./utils";
+import * as cookieParser from "cookie-parser";
+
+//Logged in user is Jessica Williamson for admin routes /teams and /teams/voyages/:voyageid
+//Logged in user is Dan ko for team member routes /teams/:teamid and /teams/:teamid/members
+//Dan Ko is part of the team with team id 4
+
+describe("Teams Controller (e2e)", () => {
+    let app: INestApplication;
+    let prisma: PrismaService;
+    let accessToken: any;
+
+    beforeAll(async () => {
+        const moduleFixture: TestingModule = await Test.createTestingModule({
+            imports: [AppModule],
+        }).compile();
+
+        await seed();
+        app = moduleFixture.createNestApplication();
+        prisma = moduleFixture.get<PrismaService>(PrismaService);
+        app.useGlobalPipes(new ValidationPipe());
+        app.use(cookieParser());
+        await app.init();
+    });
+
+    afterAll(async () => {
+        await prisma.$disconnect();
+        await app.close();
+    });
+
+    describe("GET /teams - gets all voyage teams", () => {
```

Comment:
could you please add a test for non admin user access for this route? (should return 403)
Looks like it's also not on swagger, if you don't mind could you add that to swagger as well?

Reply:
Ok , I will add the test for non admin user and update the swagger responses as well.